Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -196,11 +196,11 @@ private bool canReadOne(string resource)
                 case Resource.HOUSING:
                     {
                         // The members of the apartment application can only read their application
-                        var housingService = new HousingService(new UnitOfWork());
-                        var sess_cde = Helpers.GetCurrentSession().ToString();
+                        HousingService housingService = new HousingService(new UnitOfWork());
+                        string sess_cde = Helpers.GetCurrentSession().SessionCode;
```

Comment:
Just curious, what's the difference between '.SessionCode' and the one without it?

Reply:
When you use `.ToString()`, what you get is a string that literally says `"Gordon360.Models.ViewModels.SessionViewModel"`

So it is literally taking an instance of a class and trying to turn it into a string, rather than getting the `SessionCode` variable from the object